orchestrator-salt/src/main/resources/salt/salt/postgresql/root-certs.sls

```diff
@@ -1,6 +1,6 @@
 {%- from 'postgresql/settings.sls' import postgresql with context %}
 
-{% if postgresql.ssl_enabled == True %}
+{% if postgresql.root_certs_enabled == True %}
```

Comment:
will it cause any issue in case of upgrades?

Reply:
Should not if all the Salt pillars are updated along with Salt states. The cert bundle is placed on an attached disk volume that is supposed to survive node replacement, so OS upgrade shall not be an issue either. If the cluster uses DB SSL before the upgrade, the cert bundle shall already exist before the upgrade is started, so this state will ideally be a no-op.

The only scenario of interest would be the migration of legacy clusters from non-SSL to SSL, but again that would require updating all the Salt pillars & states together.

Reply:
Note: The dict `postgresql` and its entry `root_certs_enabled` are initialized in `postgresql/settings.sls` using Jinja2, so in the worst case this value would just evaluate to `False` in the absence of the proper Salt pillar.